apiserver/uniter/uniter.go

```diff
@@ -1689,11 +1707,75 @@ func (u *UniterAPIV3) getOneNetworkConfig(canAccess common.AuthFunc, unitTagArg,
 }
 
 // SLALevel returns the model's SLA level.
-func (u *UniterAPIV3) SLALevel() (params.StringResult, error) {
+func (u *UniterAPI) SLALevel() (params.StringResult, error) {
 	result := params.StringResult{}
 	sla, err := u.st.SLALevel()
 	if err == nil {
 		result.Result = sla
 	}
 	return result, err
 }
```

Comment:
That is correct, the only thing standing out is with NetworkConfig stuff that was 'left' in UniterAPIV3